pygin/pygin/src/panel.cpp

```diff
@@ -3,6 +3,27 @@
 #include "panel.hpp"
 #include "far_api.hpp"
 #include "py.integer.hpp"
+#include "python.hpp"
+
+#include <ranges>
+
+namespace {
+
+void WINAPI free_py(void* UserData, const struct FarPanelItemFreeInfo* Info)
+{
+	Py_XDECREF(UserData);
+}
+	
+void let_far_manage_py_items_life_time(std::ranges::subrange<PluginPanelItem*> const& items) noexcept
```

Comment:
`span` maybe?

Reply:
Fixed in 25bec961a4332cba0ae5649d2dd13af0bff6261c.